doc/09-object-types.md

```diff
@@ -742,6 +742,19 @@ to evenly distribute all checks over a certain period of time, i.e. to avoid loa
 > The `address` and `address6` attributes are required for running commands using
 > the `$address$` and `$address6$` runtime macros.
 
+> **Info**
+>
+> The macros $host.name$, $service.description$, $host.address$ or $address$ are available for forming the notes url and action url
+
+> **Info**
```

Comment:
Why a second Info section? I think this can also be placed below the first Info section. :)